kv/txn.go

```diff
@@ -17,23 +17,112 @@ package kv
 import (
 	"context"
 	"errors"
+	"fmt"
 	"math"
 	"math/rand"
+	"sync"
 	"time"
 
 	"github.com/pingcap/failpoint"
 	"github.com/pingcap/tidb/parser/terror"
 	"github.com/pingcap/tidb/util/logutil"
+	"github.com/tikv/client-go/v2/oracle"
 	"go.uber.org/zap"
 )
 
+const (
+	// TimeToPrintLongTimeInternalTxn is the duration if the internal transaction lasts more than it,
+	// TiDB prints a log message
+	TimeToPrintLongTimeInternalTxn = time.Minute * 5
```

Comment:
This seems to be a bit bigger for most cases? Not quite sure, @wjhuang2016 @tangenta @tiancaiamao any advices about it?

Reply:
sysbench test data was pasted in the PR description.
I think the tps and duaration variance is deviation